### PR TITLE
vm/server

### PR DESCRIPTION
**Pull request by dhvanilshah-tenable**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Python Version(s) Tested:
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ngaglani-tenable** [comment]
Documentation related changes are missing.

File: tenable/io/v3/vm/server/api.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+.. rst-class:: hide-signature
+.. autoclass:: ServerAPI
+    :members:
+'''
+from typing import Dict
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class ServerAPI(ExploreBaseEndpoint):
```
Comment (ngaglani-tenable) [inline tenable/io/v3/vm/server/api.py:19]:
```suggestion
class ServersAPI(ExploreBaseEndpoint):
```

**Review by ngaglani-tenable — APPROVED** [review]

**Comment by dhvanilshah-tenable** [comment]
> Documentation related changes are missing.

Added the changes.

**Review by parvdesai-tenable — APPROVED** [review]

File: tests/io/v3/vm/server/test_server_api.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        'msp': True,
+        'nessus_type': 'Nessus Cloud',
+        'nessus_ui_build': '308',
+        'nessus_ui_version': '11.0.375',
+        'notifications': [],
+        'plugin_set': '202112171226',
+        'region': 'abc',
+        'scanner_boottime': 1617702260,
+        'server_build': 'D20144',
+        'server_uuid': '1b6c59f6-0000-0000-0000-01996d3b57911c81433cc5c59a7b',
```
Comment (tomeara) [inline tests/io/v3/vm/server/test_server_api.py:52]: should probably be `id`

**Review by tomeara — CHANGES_REQUESTED** [review]

File: tests/io/v3/vm/server/test_server_api.py (hunk 2 above)
Comment (dhvanilshah-tenable) [reply tests/io/v3/vm/server/test_server_api.py:52]: @tomeara Just to confirm. This should be **server_id**. Right? Or should we only keep it as **id**?

Comment (tomeara) [reply tests/io/v3/vm/server/test_server_api.py:52]: I think we can just keep it as `id`, since the entity is `server` as is.

Comment (dhvanilshah-tenable) [reply tests/io/v3/vm/server/test_server_api.py]: Made the changes.

**Review by tomeara — APPROVED** [review]

**Pull request merged**